community/kernel/src/main/java/org/neo4j/kernel/impl/util/CoreNodeWrappingNodeValue.java

```diff
@@ -31,13 +31,17 @@
 import org.neo4j.values.virtual.NodeValue;
 import org.neo4j.values.virtual.VirtualValues;
 
-public class NodeProxyWrappingNodeValue extends NodeValue
+/**
+ * This NodeValue wraps a Core API Node and delegates method call to the
+ * Node instance.
+ */
+public class CoreNodeWrappingNodeValue extends NodeValue
```

Comment:
So even with `NodeProxy` now being a `NodeValue` we _still_ have the wrappers. Then what do we gain with this change?